interfaces/PA-App/src/app/user-menu/user-menu.component.ts

```diff
@@ -36,4 +40,64 @@ export class UserMenuComponent implements OnInit {
     this.popoverController.dismiss();
     window.location.reload();
   }
+
+  async deletePrompt() {
+    const alert = await this.alertController.create({
+      header: this.translate.instant('account.delete-account-id'),
+      inputs: [
+        {
+          name: 'password',
+          type: 'text',
+          placeholder: this.translate.instant('account.enter-password')
+        },
+      ],
+      buttons: [
+        {
+          text: this.translate.instant('shared.submit-button'),
+          handler: async data => {
+            console.log('Confirm Ok');
+            await this.deleteAccountId(data.password);
+          },
+        },
+        {
+          text: this.translate.instant('account.close'),
+          handler: () => {
+            this.alertController.dismiss();
+          }
+        }
+      ]
+    });
+    await alert.present();
+  }
+
+  public async deleteAccountId(password: string) {
+    console.log('password', password);
+    const wallet = await this.paData.retrieve(this.paData.type.wallet);
+    await this.paData.deleteAccount(password).then(
+      async () => {
+        console.log('Delete wallet', wallet);
+        await this.deleteWallet(wallet);
```

Comment:
We could trigger both requests simultaneously one a PA clicks the confirm...
Because they don't really 'depend' on each other, right?

It could be fine as it is, but having less 'daisy-chaining' in our code would be nicer, I think.